draft/draft-bensley-rpsl-exclude-members.md

```diff
@@ -224,6 +224,16 @@ The IRR software MUST NOT require that the primary key of an entry in the `excl-
 
 The IRR software MUST NOT require that the registry scope which precedes the object primary key, is a registry the IRR software knows to be a valid registry. An authoritative IRR server may have it's content mirrored to resolver IRR servers, which have visibility of many more registries.
 
+## Joint vs. Split Attributes
+
+The `excl-members` attribute could have been designed to be two separate attributes i.e., `excl-members` and `excl-mp-members`. This would provide finer grained control by only excluding a primary key found in a `members` attribute, and only excluding a primary key found in an `mp-members` attribute, respectively. This creates additional problems though:
+
+1. If the primary key of an object which needs to be excluded exists in the data hierarchy in a `members` attribute, but is only excluded if found in an `mp-members` attribute (e.g., it is only referenced in `excl-mp-members`), the object referenced is not successfully excluded. The problem is that the wrong exclude attribute has been used. This problem doesn't exist with a single combined exclude attribute.
+1. The syntax of the `mp-members` attribute allows it to contain all types of value supported by the `members` attribute, plus additional value types not supported by the `members` attribute (syntactically `mp-members` supports a superset of `members`). A primary key type which is valid in both attributes, such as a ROUTE-SET, could be included via the `members` attribute or the `mp-members` attribute in the data hierarchy. To ensure this object is excluded, it's primary key would need to be added to both exclude attributes. This creates the problem that the two attributes would need to always store all values which are valid in the `members` attribute. This would render an `excl-members` attribute redundant because everything is duplicated in an `excl-mp-members` attribute. This problem doesn't exist with a single combined exclude attribute.
```

Comment:
"ROUTE-SET` should be lower case, to be in keeping with the rest of the document